pkg/service/test/test.go

```diff
@@ -465,17 +470,21 @@ func (t *tester) testHttp(tc models.TestCase, actualResponse *models.HttpResp) (
 	noise := tc.Noise
 
 	var (
-		bodyNoise   []string
-		headerNoise = map[string]string{}
+		bodyNoise   = noiseConfig["body"]
+		headerNoise = noiseConfig["header"]
 	)
 
 	for _, n := range noise {
 		a := strings.Split(n, ".")
 		if len(a) > 1 && a[0] == "body" {
 			x := strings.Join(a[1:], ".")
-			bodyNoise = append(bodyNoise, x)
+			if _, ok := bodyNoise[x]; !ok {
+				bodyNoise[x] = []string{}
+			}
 		} else if a[0] == "header" {
-			headerNoise[a[len(a)-1]] = a[len(a)-1]
+			if _, ok := headerNoise[a[len(a)-1]]; !ok {
+				headerNoise[a[len(a)-1]] = []string{}
```

Comment:
same here , Why are we not supporting regex in noise written in test-case file too..?

Reply:
done!